scripts/makefile/front.mk

```diff
@@ -55,4 +54,4 @@ storybook:
 	@echo "Running storybook..."
 	docker pull $(IMAGE_FRONT)
 	$(call frontexec, yarn install --ignore-optional --check-files)
-	$(call frontexec, yarn storybook -p $(FRONT_PORT))
+	$(call frontexec, yarn build-storybook --verbose)
```

Comment:
++

I guess global todo is : 

1. ~~Update existing `make storybook` command to generate static files (https://github.com/skilld-labs/skilld-docker-container/blob/195-add-static-storybook/scripts/makefile/front.mk#L53)~~ Done
2. Add `make storybook` to list of job in `make all_ci` (https://github.com/skilld-labs/skilld-docker-container/blob/195-add-static-storybook/Makefile#L35)
3. Add a few commands to `.deploy_template: &deploy_template` (https://github.com/skilld-labs/skilld-docker-container/blob/195-add-static-storybook/.gitlab-ci.yml#L148), at the end of `script` part with goal to move storybook static files in a dir inside web/ (like it is done here https://github.com/skilld-labs/skilld-docker-container/blob/195-add-static-storybook/.gitlab-ci.yml#L220-L223)

That way, generating static storybook will be part of RA job and everything remains clean.
Is it ok for you ?

Reply:
@davidferlay, @andypost Have another vision